src/seedu/addressbook/ui/TextUi.java

```diff
@@ -164,5 +165,15 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
-
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+	public static  String getPrintableString(Printable... printables) {
+		String concatString = "";
+		for (Printable p : printables) {
+			concatString += p.getPrintableString() + " ";
+        }
+		return concatString.trim();
+	}
```

Comment:
A good try. What to do you think if this goes in ReadOnlyPerson interface? I think it looks more meaningful in Person context.